src/main/java/uk/gov/ccs/conclave/data/migration/service/OrganisationService.java

```diff
@@ -192,7 +192,6 @@ private Boolean checkForAdminOnNewOrg(final Organisation organisation) {
                 System.out.println(String.format("HERE -> B (userRole):  %s", userRole));
                 System.out.println(String.format("HERE -> C (userRole.getName()):  %s", userRole.getName()));
                 System.out.println(String.format("HERE -> D (userRole.getName().equals(Organisation Administrator)):  %s", userRole.getName().equals("Organisation Administrator")));
-                System.out.println(String.format("HERE -> E (userRole.isUserRoleAdmin()):  %s", userRole.isUserRoleAdmin()));
                 if (userRole.getName().equals("Organisation Administrator")) {
```

Comment:
```suggestion
                return userRole.isUserRoleAdmin()
```
 (in place of lines 196-202)